tools/tools.go

```diff
@@ -0,0 +1,19 @@
+//go:build tools
+
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package tools
+
+import (
+	_ "github.com/client9/misspell/cmd/misspell"
+	_ "github.com/google/addlicense"
+	_ "github.com/gordonklaus/ineffassign"
+	_ "github.com/maxbrunsfeld/counterfeiter/v6"
+	_ "github.com/onsi/ginkgo/ginkgo"
+	_ "golang.org/x/tools/cmd/goimports"
+	_ "google.golang.org/protobuf/cmd/protoc-gen-go"
```

Comment:
TODO add golint and gocyclo

Reply:
@mbrandenburger do you want to address this before we merge?

Reply:
added the missing ones; and replaced deprecated golint by staticchecks